### DISCUSSION TITLE
Mapsforge maps (2024-06-18 to 2024-06-28)

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#580] [Q&A] [answered]
There was a problem with the land polygons in the Mapsforge maps of **2024-06-18** to **2024-06-28**.

Please pause downloading or updating your Mapsforge maps between these dates.

We are currently rebuilding the Mapsforge maps with the correct land polygons
and will be available soon.

- https://github.com/mapsforge/mapsforge/issues/1489

In the meantime, you could also use the OpenAndroMaps or Freizeitkarte maps.

**Comment by DolomitesTom** [comment]
Including or exkluding 27.6.? Can maps from 27.6. be downloaded?

**Reply by devemux86** [reply]
@DolomitesTom 

> Can maps from 27.6. be downloaded?

No, prefer the new maps from **28 / 6**.

Map re-generation is currently running and all maps will be updated, many have already been updated.

**Reply by DolomitesTom** [reply]
But the download manager shows no maps from 28.6.

**Reply by devemux86** [reply]
For example, Germany regions are already updated (28/6).

**Comment by devemux86** [comment]
Also, it is not so important to constantly update the offline maps.

If you already have an offline map for an area, you can still use it.

**Comment by devemux86** [comment, marked as answer]
Mapsforge maps with the problem were all updated.

If you see anything else, please let me know.
